lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -26,17 +26,19 @@ function PropCore.WithinPropcoreLimits()
 	return (sbox_E2_maxProps:GetInt() <= 0 or E2totalspawnedprops<sbox_E2_maxProps:GetInt()) and E2tempSpawnedProps < sbox_E2_maxPropsPerSecond:GetInt()
 end
 
-function PropCore.ValidSpawn(ply, model, isVehicle)
+function PropCore.ValidSpawn(ply, model, isVehicle, seat)
 	local ret -- DO NOT RETURN MID-FUNCTION OR 'LimitHit' WILL BREAK
 	local limithit = playerMeta.LimitHit
 	playerMeta.LimitHit = function() end
+	
+	if seat == "" then seat == "Seat_Airboat"
 
 	if not PropCore.WithinPropcoreLimits() then
 		ret = false
 	elseif not (util.IsValidProp( model ) and WireLib.CanModel(ply, model)) then
 		ret = false
 	elseif isVehicle then
-		ret = gamemode.Call( "PlayerSpawnVehicle", ply, model, "Seat_Airboat", list.Get( "Vehicles" ).Seat_Airboat ) ~= false
+		ret = gamemode.Call( "PlayerSpawnVehicle", ply, model, seat, list.Get( "Vehicles" ).seat ) ~= false
```

Comment:
This syntax is incorrect. You're basically doing `list.Get("Vehicles")["seat"]`, while you're supposed to do `list.Get("Vehicles")[seat]` to use the `seat` variable